cmd/oras/internal/option/platform.go

```diff
@@ -31,7 +31,7 @@ type Platform struct {
 
 // ApplyFlags applies flags to a command flag set.
 func (opts *Platform) ApplyFlags(fs *pflag.FlagSet) {
-	fs.StringVarP(&opts.Platform, "platform", "", "", "fetch the manifest of a specific platform if target is multi-platform capable")
+	fs.StringVarP(&opts.Platform, "platform", "", "", "only fetch a specific platform if target is multi-platform capable, in the form of os[/arch][/variant][:os_version])")
```

Comment:
My suggestion: `only fetch a manifest with specifying a specific platform if the target is multi-platform capable, in the form of os[/arch][/variant][:os_version])"`